### PR TITLE
dev-python/sabyenc: add 5.4.2

### PR DESCRIPTION
**Pull request by joecool1029**
Will need this bumped to bring up sabnzbd to current. It shouldn't break existing sabnzbd versions. 

Signed-off-by: Joe Kappus <joe@wt.gd>

### CODE REVIEW

File: dev-python/sabyenc/sabyenc-5.4.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (thesamesam) [inline dev-python/sabyenc/sabyenc-5.4.2.ebuild:4]: 8 please

File: dev-python/sabyenc/sabyenc-5.4.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+PYTHON_COMPAT=( python3_{8..11} )
```
Comment (thesamesam) [inline dev-python/sabyenc/sabyenc-5.4.2.ebuild:6]: Can you try DISTUTILS_USE_PEP517?

Comment (joecool1029) [reply dev-python/sabyenc/sabyenc-5.4.2.ebuild:6]: ok yeah, that works.

File: dev-python/sabyenc/sabyenc-5.4.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{8..11} )
+inherit distutils-r1
+
+DESCRIPTION="Module providing raw yEnc encoding/decoding for SABnzbd"
+HOMEPAGE="https://github.com/sabnzbd/sabyenc"
+SRC_URI="https://github.com/sabnzbd/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```
Comment (arthurzam) [inline dev-python/sabyenc/sabyenc-5.4.2.ebuild:12]: Please use `.gh.tar.gz` for python tarballs from github :)
```suggestion
SRC_URI="
	https://github.com/sabnzbd/${PN}/archive/v${PV}.tar.gz
		-> ${P}.gh.tar.gz
"
```

File: dev-python/sabyenc/sabyenc-5.4.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+LICENSE="LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+BDEPEND="test? (
+		dev-python/chardet[${PYTHON_USEDEP}]
+	)"
```
Comment (arthurzam) [inline dev-python/sabyenc/sabyenc-5.4.2.ebuild:20]: This looks out of style for python packages :)
```suggestion
BDEPEND="
	test? (
		dev-python/chardet[${PYTHON_USEDEP}]
	)
"
```

**Review by arthurzam — CHANGES_REQUESTED** [review]

**Pull request closed without merging**